source/source_test.go

```diff
@@ -0,0 +1,186 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package source
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/conduitio-labs/conduit-connector-mongo/config"
+	"github.com/conduitio-labs/conduit-connector-mongo/source/mock"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/golang/mock/gomock"
+	"github.com/matryer/is"
+)
+
+func TestSource_Configure_success(t *testing.T) {
+	t.Parallel()
+
+	is := is.New(t)
+
+	s := Source{}
+
+	err := s.Configure(context.Background(), map[string]string{
+		config.KeyURI:        "mongodb://localhost:27017",
+		config.KeyDB:         "test",
+		config.KeyCollection: "users",
+	})
+	is.NoErr(err)
+	is.Equal(s.config, Config{
```

Comment:
nitpick: lets have the expected `Config` as a variable, and assert that `is.Equal(s.config, got)` instead, this would just make it easier to read and more tidy

Reply:
Sounds reasonable. Thank you!